packages/react-component-library/src/components/Timeline/Timeline.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+
+import { getKey } from './helpers'
+import { TimelineProvider } from './context'
+
+import { Side, Row, TodayMarker, Months, Weeks } from '.'
+
+export interface TimelineProps extends ComponentWithClass {
+  rowData: any[]
+  startDate?: Date
+}
+
+export const Timeline: React.FC<TimelineProps> = ({ rowData, startDate }) => {
+  return (
+    <TimelineProvider startDate={startDate}>
+      <article className="timeline">
+        <Side rowData={rowData} />
+        <div className="timeline__inner">
+          <header className="timeline__header">
+            <TodayMarker />
+            <Months />
+            <Weeks />
+          </header>
+          <main className="timeline__main">
+            {!rowData ||
```

Comment:
It feels like a variable could be extracted here which is used here and on lines 30-31. It would help with readability.